plugin/wakatime.vim

```diff
@@ -6,7 +6,7 @@
 " Website:     https://wakatime.com/
 " ============================================================================
 
-let s:VERSION = '11.2.0'
+let s:VERSION = '11.3.0'
```

Comment:
I prefer bumping versions in separate commits/PRs not bundled with feature changes.